### ISSUE TITLE
First instructions in README.md doesn't use yarn for install

### ISSUE THREAD

**Issue by fredvd** [plone/volto#672]
First quick install instructions are a bit typical in the README.md

If yarn is the preferred package manager, instead of 

```
$ npm install -g yarn
$ npm install -g @plone/create-volto-app
$ create-volto-app myvoltoproject
$ cd myvoltoproject
```
I'd expect using yarn to install create-volto-app like:

```
$ npm install -g yarn
$ yarn global add @plone/create-volto-app
$ create-volto-app myvoltoproject
$ cd myvoltoproject
```

**Comment by tisto** [comment]
@fredvd I am afraid it is a bit more complex than that. A common practice is to use npm for globally installed packages and yarn for locally installed packages. I get your point. Though it is hard to make a final decision on this.

**Comment by sneridagh** [comment]
@fredvd unfortunatelly use npm here (when the package install local commands) is required. Maybe we can explain briefly the reasons in the README, and more detailed in the docs.

**Comment by sneridagh** [comment]
Did `yarn global add @plone/create-volto-app` worked for you?

**Comment by fredvd** [comment]
Thank you for the feedback. I didn't realise until now that yarn global is doing things different from npm -g.  

@sneridagh Yes, "yarn global add @plone/create-volto-app" worked for me, but that might be because I'm using nodeenv to 'really' isolate' node/npm/yarn environments from each other.  This might 'fix' or hide yarn problems, so I didn't notice. :-(

I still can't wrap my head around why there hasn't been stronger demand for something like virtualenv in the js community. I realise nodeenv is another niche solution, but it does make me feel a bit safer about not poluting my global npm environment.

**Comment by fredvd** [comment]
Nope, it has nothing to do with nodeenv, it accidentally worked because I probably played around with yarn last year and had added $HOME/.yarn/bin to my path.  yarn global is definitely not safe.

```
fred@Tycho:volto $ source bin/activate
(volto) fred@Tycho:volto $ which create-volto-app 
/Users/fred/.yarn/bin/create-volto-app

(volto) fred@Tycho:volto $ which node
/Users/fred/test/volto/bin/node
```
node is contained and served from the activated nodeenv/virtualenv, create-volto-app was not.

**Comment by sneridagh** [comment]
@fredvd 

I personally only install globally those tools that have a command associated, usually utilities that create boilerplates, like `create-react-app` and `@plone/create-volto-app` or general utilities (like `npx`). For the rest I always use local installs (even for the most insignificant package). If you need to call an utility command in the local `node_modules` you can use the helper `npx`.

https://www.npmjs.com/package/npx

So I have always my node env "clean", same approach that with normal system Python. Then I use `nvm` for manage installed (and active) NodeJS versions.

**Comment by sneridagh** [comment]
Closing, already merged PR #688
